orchagent/portsorch.cpp

```diff
@@ -1860,6 +1867,24 @@ void PortsOrch::initPortSupportedSpeeds(const std::string& alias, sai_object_id_
     m_portStateTable.set(alias, v);
 }
 
+void PortsOrch::initPortCapAutoNeg(Port &port)
+{
+    sai_status_t status;
+    sai_attribute_t attr;
+
+    attr.id = SAI_PORT_ATTR_SUPPORTED_AUTO_NEG_MODE;
+    status = sai_port_api->get_port_attribute(port.m_port_id, 1, &attr);
+    if (status == SAI_STATUS_SUCCESS)
+    {
+        port.m_cap_an = attr.value.booldata ? 1 : 0;
+    }
+    else
+    {
+        port.m_cap_an = 1; /* Default to 1 for vstest */
```

Comment:
Not sure about this, it means that m_cap_an will be 1 even if get_port_attribute returns error.

Reply:
Yes, this is intended to avoid test failure in vstest

Reply:
So, what if get_port_attribute actually return some error in production environment?

Reply:
It ends up with the AN test failures, and hence the code coverage will be reduced.
An alternative solution is to extend the SAI model in the docker-sonic-vs